### PR TITLE
Add documentation for lib/s390base.pm

### PR DESCRIPTION
**Pull request by Zaoliang**
see https://progress.opensuse.org/issues/57428

### CODE REVIEW

File: lib/s390base.pm [hunk 1]
```diff
@@ -17,6 +22,13 @@ use utils;
...
 
+=head2 copy_testsuite
+
+ copy_testsuite();
+
+Fetch the testcase and common libraries from openQA server
+
+=cut
 sub copy_testsuite {
     my ($self, $tc) = @_;
```
Comment (SergioAtSUSE) [inline lib/s390base.pm]: There is a parameter here `$tc`, this needs to be mentioned in the documentation.

File: lib/s390base.pm [hunk 2]
```diff
@@ -37,13 +49,29 @@ sub copy_testsuite {
...
 
+=head2 execute_script
+
+ execute_script();
+
+Execute script and upload script logs
+
+=cut
 sub execute_script {
     my ($self, $script, $scriptargs, $timeout) = @_;
```
Comment (SergioAtSUSE) [inline lib/s390base.pm]: There are parameters here `$script, $scriptargs, $timeout`, this needs to be mentioned in the documentation.

File: lib/s390base.pm [hunk 3]
```diff
@@ -37,13 +49,29 @@ sub copy_testsuite {
...
+ cleanup_testsuite();
+
+Currently it just returns value 1. 
+
+It should be able to remove files saved at ./tmp
+
+=cut
 sub cleanup_testsuite {
     return 1;
     # FIXME assert_script_run 'cd / && rm -rf ./tmp';
```
Comment (SergioAtSUSE) [inline lib/s390base.pm]: :man_facepalming: how could this end up here?

Comment (Zaoliang) [reply lib/s390base.pm]: Actually this just returns 1. 
assert_script_run 'cd / && rm -rf ./tmp'; is not working I'm afraid. see #FIXME

Comment (Zaoliang) [reply lib/s390base.pm]: okay, just see your PR :)

File: lib/s390base.pm [hunk 4]
```diff
@@ -17,6 +22,15 @@ use utils;
 use strict;
 use warnings;
 
+=head2 copy_testsuite
+
+ copy_testsuite();
```
Comment (SergioAtSUSE) [inline lib/s390base.pm]:
```suggestion
 copy_testsuite($tc);
```

File: lib/s390base.pm [hunk 5]
```diff
@@ -37,13 +51,30 @@ sub copy_testsuite {
     save_screenshot;
 }
 
+=head2 execute_script
+
+ execute_script();
```
Comment (SergioAtSUSE) [inline lib/s390base.pm]:
```suggestion
 execute_script($script, $scriptargs, $timeout);
```

File: lib/s390base.pm [hunk 6]
```diff
@@ -17,6 +22,15 @@ use utils;
 use strict;
 use warnings;
 
+=head2 copy_testsuite
+
+ copy_testsuite($tc);
+
+Fetch the testcase and common libraries from openQA server.
+The parameter '$tc' name of testcase and common library.
```
Comment (SergioAtSUSE) [inline lib/s390base.pm]: I am taking the POD of **assert_screen** for comparison.

```suggestion
The parameter C<$tc> name of testcase and common library. 
```

**Pull request merged**